more/free-programming-interactive-tutorials-en.md

```diff
@@ -107,6 +107,7 @@
 * [AWS Identity and Access Management (IAM)](https://run.qwiklabs.com/focuses/7782?catalog_rank=%7B%22rank%22%3A6%2C%22num_filters%22%3A1%2C%22has_search%22%3Atrue%7D&parent=catalog&search_id=3605942) - *registration required*
 * [AWS Lambda](https://run.qwiklabs.com/focuses/6431?catalog_rank=%7B%22rank%22%3A2%2C%22num_filters%22%3A1%2C%22has_search%22%3Atrue%7D&parent=catalog&search_id=3605949) - *registration required*
 * [AWS Simple Storage Service (S3)](https://run.qwiklabs.com/focuses/7860?catalog_rank=%7B%22rank%22%3A3%2C%22num_filters%22%3A0%2C%22has_search%22%3Atrue%7D&parent=catalog&search_id=3597563) - *registration required*
+* [Cloud Computing Tutorial by Scaler](https://www.scaler.com/topics/cloud-computing/) - *free course*
```

Comment:
```suggestion
* [Cloud Computing Tutorial by Scaler](https://www.scaler.com/topics/cloud-computing/) - Scaler Topics
```
For this and all the added enties, there's no need to add a notation that it's free - all the courses listed here should be free